src/views/Inventory.vue

```diff
@@ -171,22 +172,16 @@ export default defineComponent({
         this.getTemporalExpr(this.getJobStatus(this.jobEnums[enumId]))?.description :
         this.$t('Disabled')
     },
-    async fetchMoreJobs() {
-      await this.store.dispatch("job/fetchMoreJobs", {
-        "inputFields":{
-          "enumTypeId": "INVENTORY_SYS_JOB",
-        },
-      });
+    getMoreJobs(jobEnums: any, enumTypeId: string) {
```

Comment:
Why this is needed? we could directly use moreJobs(jobEnums, enumTypeId)

Reply:
Yes sir. Fixed it.